samples/dictionary.sample.yml

```diff
@@ -0,0 +1,763 @@
+# Key to Entry Fields
+#
+# Each object represents a *dictionary entry*.
+#
+# category:                                         # The lexical category for the entry. This is displayed to the end user as-is, but it is also used by the dictionary app for ...? For Plains Cree, the value of this field is whatever is in Arok's `\ps` field in the Toolbox file. For now, each entry may only have one lexical category.
+# FST:                                              # Object containing information for the FST.
+#   lemma:                                          # The lemma used by the FST, in the proto-orthography.
+#   stem:                                           # The stem used by the FST, in the proto-orthography.
+# head:                                             # The head(word|morpheme|phrase) displayed to the user, in the proto-orthography. Will get transliterated to other orthographies in the dictionary app depending on which orthography the user selects.
+# key:                                              # A unique key for this entry. Includes a trailing homograph number if the form is a homograph.
+# paradigm: <static> | <dynamic> | "unknown" | null # Indicates the paradigm used by this entry. null = non-inflecting / don't show a paradigm. "unknown" indicates that additional work is needed by linguists---don't show a paradigm (or maybe show "unknown paradigm" or something).
+# POS: N | V | Part | Pro | (others?)               # The general word class / part of speech. Displayed to the user as-is. Used by the dictionary app for emojis and ...?
+# senses:                                           # An array of senses / definitions for this entry.
+#   - definition:                                   # The definition for this sense.
+#     sources:                                      # An array of data sources for this sense.
+
```

Comment:
This should support the option of only choosing a subset of the sources for display. That is, that the provided definition can be considered accurate for any of the sources, even if only one is chosen for display. Otherwise, more structure might be needed.